src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final String COMMA = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result,";
+    public static final String NEW_LINE = System.lineSeparator();
+    public static final int ONE = 1;
+    public static final String regex = "[\\[\\] ,]";
+    private int result;
+    private int sumSupply;
+    private int sumBuy;
+    private String[] arrayOfList;
+    private StringBuilder calculationResult = new StringBuilder();
```

Comment:
All this vars can be local
![image](https://user-images.githubusercontent.com/20326858/219005208-1620ca61-67fd-408b-8ff4-158b5ee8a424.png)